webpack.config.js

```diff
@@ -13,7 +14,8 @@ module.exports = {
     library: '[name]'
   },
   plugins: [new HtmlWebpackPlugin({
-    template: 'app/index.ejs'
+    // previously needed for default truffle app
+    // template: 'app/index.ejs'
```

Comment:
If we are forgoing the HTML thing and relying on terminal based tests for this repo, I think that it would make sense to adjust the package.json